apps/browser-tests/src/App.tsx

```diff
@@ -31,9 +34,11 @@ async function createAndPreparePsbt(psbtParams: PSBTBuilderOptions) {
 function Transactions({
   provider,
   connectedAddresses,
+  satsConnectWallets,
```

Comment:
where are we passing this? this is not being set

Reply:
Missed this out! Fixed!